scripts/pre_push_hook.py

```diff
@@ -200,6 +221,13 @@ def _start_sh_script(scriptname):
     return task.returncode
 
 
+def _is_in_dirty_state():
+    """Returns true if the repos state is dirty (modified files are uncommited)
+    ignores untracked files."""
+    state = subprocess.check_output(GIT_IS_DIRTY_CMD.split(" "))
```

Comment:
Sure,  I was thinking about how to handle untracked files. 

Right now untracked files are filtered from the status command but in principle this might interfere when changing branches (although the probability is rather low) 

For the sake of usability I'm ignoring those for now,  would that be ok?